### PR TITLE
Add imc feedback banner

### PR DESCRIPTION
**Pull request by ConalCosgrove**
If environment variables are set, adds a banner to the top of IMC room containing links to feedback outlets for users.
Variables: 
- REFOCUS_ROOMS_CONTACT_EMAIL
- REFOCUS_ROOMS_CHATTER_URL

Screenshot 2019-07-25 at 17 15 00

### CODE REVIEW

File: view/rooms/app.js [hunk 1]
```diff
@@ -1074,10 +1090,11 @@ window.onload = () => {
     debugMessage(`Error ${err}`);
   })
   .then((res) => {
-    if(res) {
+    if (res) {
       _roomTypeName = res.body.name;
       const subTitle = `${_roomName} - ${_roomTypeName}`;
       uPage.setSubtitle(subTitle);
+      uPage.setBannerText(banner);
```
Comment (iamigo) [inline view/rooms/app.js]: Should this be inside an `if (banner) { ... }` ?

File: config.js [hunk 2]
```diff
@@ -335,6 +339,8 @@ module.exports = {
   pubSubStatsLoggingInterval,
   queueTime95thMillis: pe.QUEUESTATS_95TH_WARNING_MILLIS,
   readReplicas,
+  refocusRoomsChatterUrl,
```
Comment (iamigo) [inline config.js]: Maybe this should be

```
refocusRoomsFeedback: {
  chatter: refocusRoomsChatterUrl,
  email: refocusRoomsContactEmail,
},
```

It makes it a little clearer and cleaner, I think.

File: view/loadView.js [hunk 3]
```diff
@@ -182,6 +181,7 @@ function getRedirectURI(viewKey, reqUrl) {
 } // This function is temporary - remove when separate deployment has settled
 
 function loadView(app, passport) {
+  console.log('loadView', refocusRoomsFeedback);
```
Comment (BrandonMurnane) [inline view/loadView.js]: console log

File: view/rooms/app.js [hunk 4]
```diff
@@ -59,6 +59,11 @@ const BOT_REQ_HEADERS = {
   Expires: '-1',
   'Cache-Control': 'private, max-age=31536000', // 31536000s = 1 year
 };
+
+const openInNewTabProps = 'rel="noopener noreferrer" target="_blank"';
+let _refocusRoomsChatterUrl;
+let _refocusRoomsContactEmail;
+let banner;
```
Comment (BrandonMurnane) [inline view/rooms/app.js]: couldnt you just set as null here and then avoid the else line 1052

File: config.js [hunk 5]
```diff
@@ -335,6 +339,10 @@ module.exports = {
   pubSubStatsLoggingInterval,
   queueTime95thMillis: pe.QUEUESTATS_95TH_WARNING_MILLIS,
   readReplicas,
+  refocusRoomsFeedback: {
+    chatter: refocusRoomsChatterUrl,
```
Comment (cassiodias) [inline config.js]: wouldn't be better ```chatter: pe.REFOCUS_ROOMS_CHATTER_URL || null``` + email and removing line 75, 76, 77 & 78

File: view/rooms/app.js (hunk 4 above)
Comment (cassiodias) [reply view/rooms/app.js]: +1

**Comment by iamigo** [comment]
ok to merge once https://github.com/salesforce/refocus/pull/1207 (fix the flappers) is merged

**Pull request merged**